src/i18n/Data/Intl/IntlLocales.php

```diff
@@ -1605,9 +1605,11 @@ public function validate($locale)
         if (!$lang || !$region) {
             return false;
         }
-        return strcasecmp($lang, $region)
-            && strcasecmp($lang, $locale)
-            && strcasecmp($region, $locale);
+        $localeCode = strtolower($lang) . '_' . strtoupper($region);
+        $locales = $this->getLocales();
+
+        return array_key_exists($localeCode, $locales)
```

Comment:
This code means that if a locale is valid on the current system, but we haven't added it to our config, we consider it invalid. We should probably not fail an otherwise valid locale just because of this.